### PR TITLE
Updated examples to include valid schema for AMP in Google Search

### PR DESCRIPTION
**Pull request by Meggin**

### CODE REVIEW

File: _get_started/create/basic_markup.md [hunk 1]
```diff
@@ -19,11 +19,31 @@ Copy this and save it to a file with a .html extension.
...
-        "headline": "Open-source framework for publishing content",
-        "datePublished": "2015-10-07T12:02:41Z",
-        "image": [
-          "logo.jpg"
-        ]
+        "mainEntityOfPage": "http://cdn.ampproject.org/article-metadata.html",
+        "headline": "Lorem Ipsum",
+        "datePublished": "1907-05-05T12:02:41Z",
+        "dateModified": "1907-05-05T12:02:41Z",
+        "description": "The Catiline Orations continue to begule engineers and designers alike -- but can it stand the test of time?",
```
Comment (rudygalfi) [inline _get_started/create/basic_markup.md]: beguile ?

File: _guides/discovery.md [hunk 2]
```diff
@@ -47,11 +47,31 @@ Example:
...
-    "headline": "Open-source framework for publishing content",
-    "datePublished": "2015-10-07T12:02:41Z",
-    "image": [
-      "logo.jpg"
-    ]
+    "mainEntityOfPage": "http://cdn.ampproject.org/article-metadata.html",
+    "headline": "Lorem Ipsum",
+    "datePublished": "1907-05-05T12:02:41Z",
+    "dateModified": "1907-05-05T12:02:41Z",
+    "description": "The Catiline Orations continue to begule engineers and designers alike -- but can it stand the test of time?",
```
Comment (rudygalfi) [inline _guides/discovery.md]: beguile ?

**Comment by rudygalfi** [comment]
Spelling nit in two places.

Otherwise, this looks like what we need. I didn't run the changes through the structured data testing tool specifically, but assume that happened?

**Comment by Meggin** [comment]
Just ran it through the SDT tool and checks are 'all good'.

**Comment by cramforce** [comment]
I don't think we should do this. This adds so much complicated stuff to our examples that has nothing to do with AMP. Lets revert the basic_markup.md change. The other changes are fine.

**Comment by Meggin** [comment]
OK.

**Comment by Meggin** [comment]
Done. Only reverted basic_markup.md changes though.

**Pull request merged**